cache_utils/utils.py

```diff
@@ -64,11 +65,25 @@ def _func_info(func, args):
     return name, args[1:]
 
 
-def _cache_key(func_name, func_type, args, kwargs):
+def _cache_key(func_name, func_type, args, kwargs, filter_args_kwargs=None):
     """ Construct readable cache key """
+    if filter_args_kwargs and inspect.isfunction(filter_args_kwargs):
```

Comment:
Does `callable` work here?

Reply:
I guess that's less restrictive, we can go with that.